doc/classes/Color.xml

```diff
@@ -19,123 +19,17 @@
 		<method name="Color">
 			<return type="Color">
 			</return>
-			<argument index="0" name="from" type="String">
-			</argument>
-			<description>
-				Constructs a color from an HTML hexadecimal color string in RGB or RGBA format. See also [method @GDScript.ColorN].
-				[codeblocks]
-				[gdscript]
-				# Each of the following creates the same color RGBA(178, 217, 10, 255).
-				var c3 = Color("#b2d90a") # RGB format with "#".
-				var c4 = Color("b2d90a") # RGB format.
-				var c1 = Color("#b2d90aff") # RGBA format with "#".
-				var c2 = Color("b2d90aff") # RGBA format.
-				[/gdscript]
```

Comment:
Missing constructor.

There's no `Color::Color(String)` yet to bind, and we discussed with core devs today that we should likely add one that can support both creating a color from HTML hexadecimal code (when prefixed with `#`), and from named colors (when not prefixed with `#`).